Source/WebKit/UIProcess/glib/WebProcessProxyGLib.cpp

```diff
@@ -72,24 +71,4 @@ void WebProcessProxy::sendMessageToWebContext(UserMessage&& message)
     sendMessageToWebContextWithReply(WTFMove(message), [](UserMessage&&) { });
 }
 
-void WebProcessProxy::platformSuspendProcess()
-{
-    auto id = processID();
-    if (!id)
-        return;
-
-    RELEASE_LOG(Process, "%p - [PID=%i] WebProcessProxy::platformSuspendProcess", this, id);
-    kill(id, SIGSTOP);
 }
-
-void WebProcessProxy::platformResumeProcess()
-{
-    auto id = processID();
-    if (!id)
-        return;
-
-    RELEASE_LOG(Process, "%p - [PID=%i] WebProcessProxy::platformResumeProcess", this, id);
-    kill(id, SIGCONT);
-}
```

Comment:
Can we leave this with just notImplemented() and a FIXME comment explaining why this is not implemented and url to the bugs or whatever?